crates/tests-e2e/src/maker.rs

```diff
@@ -29,8 +29,10 @@ impl Maker {
         Self::new(client, "http://localhost:18000")
     }
 
+    /// Check whether maker is running and that it is connected to all services
+    /// it depends on
     pub async fn is_running(&self) -> bool {
-        self.get("/").await.is_ok()
+        self.get("/health").await.is_ok()
```

Comment:
Do we want to interpret the healthcheck result here? Or is it always okay if it returns 200?

Reply:
added interpreting of the result.